providers/oidc.go

```diff
@@ -181,7 +181,9 @@ func (p *OIDCProvider) redeemRefreshToken(ctx context.Context, s *sessions.Sessi
 		return fmt.Errorf("failed to get token: %v", err)
 	}
 
-	newSession, err := p.createSession(ctx, token, true)
+	idString := "notfound"
```

Comment:
Can you explain this one, I don't understand where this came from or why?

Reply:
no idea - it probably looks like its left over from debugging. tbh its been 3 months since i looked at this, so I am going to have to go back through and figure out what i was doing